test/integration/xds_integration_test.cc

```diff
@@ -36,5 +40,49 @@ TEST_P(XdsIntegrationTest, RouterRequestAndResponseWithBodyNoBuffer) {
   testRouterRequestAndResponseWithBody(1024, 512, false);
 }
 
+typedef HttpProtocolIntegrationTest LdsIntegrationTest;
+
+INSTANTIATE_TEST_SUITE_P(Protocols, LdsIntegrationTest,
+                         testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams(
+                             {Http::CodecClient::Type::HTTP1}, {FakeHttpConnection::Type::HTTP1})),
+                         HttpProtocolIntegrationTest::protocolTestParamsToString);
+
+// Sample test making sure our config framework correctly reloads listeners.
```

Comment:
Yeah, this is already somewhat covered by https://github.com/envoyproxy/envoy/blob/master/test/integration/ads_integration_test.cc#L658 and other xDS integration tests, but if the idea is to show off the new config helper features, this works.